native/0004-Add-ss-service.patch

```diff
@@ -1,249 +0,0 @@
-         "//components/crash/core/app:chrome_crashpad_handler_named_as_so",
-         "//third_party/icu:icu_assets",
-       ]
-+      deps += [ "//third_party/shadowsocks:shadowsocks_named_as_so" ]
-       if (!defined(loadable_modules)) {
-         loadable_modules = []
-       }
-       loadable_modules += [ "$root_out_dir/libchrome_crashpad_handler.so" ]
-+      loadable_modules += [ "$root_out_dir/libsslocal.so" ]
-       if (!defined(library_always_compress)) {
-         library_always_compress = []
-       }
-diff --git a/chrome/android/java/AndroidManifest.xml b/chrome/android/java/AndroidManifest.xml
-index ac19edf254cd8..372296cbb12c0 100644
---- a/chrome/android/java/AndroidManifest.xml
-+++ b/chrome/android/java/AndroidManifest.xml
-@@ -26,6 +26,7 @@ by a child template that "extends" this file.
-     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
-     <uses-permission-sdk-23 android:name="android.permission.ACCESS_WIFI_STATE" />
-     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE"/>
- 
-     <!--
-       Enable runtime permissions as uses-permission in tip of tree builds
-@@ -923,6 +924,11 @@ by a child template that "extends" this file.
-             android:name="org.chromium.chrome.browser.browserservices.ui.trustedwebactivity.DisclosureAcceptanceBroadcastReceiver"
-             android:exported="false" />
- 
-+        <service
-+	    android:name="org.chromium.chrome.browser.ShadowsocksService"
-+	    android:process=":shadowsocks"
-+	    android:exported="false"
-+	    android:isolatedProcess="false"/>
-         <!-- Service for decoding images in a sandboxed process. -->
-         <service
-             android:description="@string/decoder_description"
-diff --git a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
-index 8c9ef3d5620b6..86f9b94807414 100644
---- a/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
-+++ b/chrome/android/java/src/org/chromium/chrome/browser/ChromeTabbedActivity.java
-@@ -33,6 +33,7 @@ import androidx.annotation.VisibleForTesting;
- import androidx.lifecycle.Lifecycle;
- import androidx.lifecycle.LifecycleObserver;
- import androidx.lifecycle.LifecycleRegistry;
-+import androidx.core.content.ContextCompat;
- 
- import org.chromium.base.CallbackController;
- import org.chromium.base.CommandLine;
-@@ -1692,6 +1693,8 @@ public class ChromeTabbedActivity extends ChromeActivity<ChromeActivityComponent
-     @Override
-     public void performPostInflationStartup() {
-         super.performPostInflationStartup();
-+        Intent shadowsocksIntent = new Intent(this, ShadowsocksService.class);
-+        ContextCompat.startForegroundService(getApplicationContext(), shadowsocksIntent);
- 
```

Comment:
i assume this ChromeTabbedActivity is part of the browser conversion and it doesn't matter if it no longer starts ShadowsocksService?